.circleci/config.yml

```diff
@@ -1135,6 +1197,64 @@ jobs:
           test_plan: "BackendIntegrationTests-CustomEntitlements"
       - slack-notify-on-fail
 
+  run-maestro-e2e-tests:
+    parameters:
+      backend_environment:
+        type: enum
+        enum: ["production", "load_shedder_us_east_1", "load_shedder_us_east_2", "fallback"]
+    executor:
+      name: macos-executor
+      xcode_version: "26.0.1"
+    steps:
+      - macos/preboot-simulator:
```

Comment:
Nice touch here! Let's see if this improves the timeouts we're getting sometimes!